Code/Tools/RemoteConsole/Core/RemoteConsoleCore.cpp

```diff
@@ -66,13 +69,20 @@ bool RCON_IsRemoteAllowedToConnect(const AZ::AzSock::AzSocketAddress& connectee)
         addresses.push_back("127.0.0.1");
     }
 
+#if defined(CARBONATED)
+    for (const AZStd::string& address : addresses)
+    {
+        AZStd::regex re(address);
+        if (address == connectee.GetIP() || AZStd::regex_match(connectee.GetIP(), re))
```

Comment:
is the string malformed? I dont follow what this is for

Reply:
The address can be a real IP as 127.0.0.1, it is processed in operator ==, the address can be a regex like a mask (but it is Java regex), it is processed in regex_match,

This is script's PR which is obsolete, please move to the real one: 
https://github.com/carbonated-dev/o3de/pull/221

Reply:
Please close this PR if it is obsolete and is no longer needed.